2011-02-11  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/generic/ldsodefs.h: include <gnu/option-groups.h>.

---
 sysdeps/generic/ldsodefs.h |    1 +
 1 file changed, 1 insertion(+)

--- a/sysdeps/generic/ldsodefs.h
+++ b/sysdeps/generic/ldsodefs.h
@@ -39,6 +39,7 @@
 #include <hp-timing.h>
 #include <tls.h>
 #include <kernel-features.h>
+#include <gnu/option-groups.h> /* For __OPTION_EGLIBC_RTLD_DEBUG.  */
 
 __BEGIN_DECLS
 
